toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -497,11 +497,9 @@ func PopulateInstallRoot(installChroot *safechroot.Chroot, packagesToInstall []s
 		generateContainerManifests(installChroot)
 	}
 
-	if config.IsKickStartBoot {
-		err = configuration.ConfigureNetwork(installChroot, config)
-		if err != nil {
-			return
-		}
+	err = configuration.ConfigureNetwork(installChroot, config)
```

Comment:
Noticed the comment `// ConfigureNetwork performs network configuration during the kickstart unattended installation process` on ConfigureNetwork() could use an update.

Reply:
Also the `### Networks` section of imageconfig.md.

Reply:
Good catch, Fixed!